tests/templates/test_integration.py

```diff
@@ -665,6 +669,11 @@ def test_qubit_result_is_wire_label_independent(self, template, diffable, nondif
 
         # merge differentiable and non-differentiable arguments:
         # we don't need them separate here
+        # Extract keys and items
+        if template.__name__ == "AmplitudeEmbedding":
+            diffable = diffable.copy()
+            diffable["features"] = np.array(diffable["features"])
+
```

Comment:
Not sure I understand?

Reply:
Ah, `features` is defined as a list in the test data for `AmplitudeEmbedding`, but `QubitStateVector` only wants an array in non-tape mode.

I'm actually not sure why this (a) doesn't cause an error in earlier tests, and (b) wasn't erroring before this PR (since it is independent of tape mode)

Reply:
Cool, we should take a decision on whether Operations and Templates are strict on this or not. At the moment there is a mixture.

Reply:
In tape mode, operation type checking is now turned off. So this only affects non tape mode